src/snowcli/utils.py

```diff
@@ -568,13 +568,24 @@ def convert_resource_details_to_dict(function_details: SnowflakeCursor) -> dict:
     return function_dict
 
 
-def generate_deploy_stage_name(name: str, input_parameters: str) -> str:
-    return name + input_parameters.replace("(", "",).replace(")", "",).replace(
-        " ",
-        "_",
-    ).replace(
-        ",",
-        "",
+def generate_deploy_stage_name(identifier: str) -> str:
+    return (
+        identifier.replace(
+            "(",
+            "_",
+        )
+        .replace(
+            ")",
+            "",
+        )
+        .replace(
+            " ",
+            "_",
+        )
+        .replace(
+            ",",
+            "",
+        )
```

Comment:
For `procedure(a int, b string)` function returned `procedurea_int_b_string` so I changed replace `(` to `_` to get `procedure_a_int_b_string`

Reply:
Let's capture this as a breaking change. Users won't be able to iteract with previously created objects.

Reply:
Added